src/Microsoft.VisualStudio.Services.Agent/AdditionalMaskingRegexes.cs

```diff
@@ -30,11 +30,13 @@ public static partial class AdditionalMaskingRegexes
         // the user, and the ":" and skip them. Similarly, it uses
         // a zero-width positive lookahead to find the "@".
         // It only matches on the password part.
-        private static string urlSecretPattern
-            = "(?<=//[^:/?#\\n]+:)" // lookbehind
-            + urlMatch       // actual match
-            + "(?=@)";              // lookahead
+        private const string lookBehind = "//[^:/?#\\n]+:";
+        private const string lookAhead = "@";
+        public static string UrlSecretPattern { get; } = $"(?<={lookBehind}){urlMatch}(?={lookAhead})";
 
-        public static string UrlSecretPattern => urlSecretPattern;
+        // Microsoft.Security.Utilities.Core SecretMasker uses NonBacktracking
```

Comment:
>uses NonBacktracking [](http://example.com/codeflow?start=58&length=20)

is this wording/casing intended? s/be simply `uses a non-backtracking` maybe?

Reply:
Fixed.